signer/src/config/mod.rs

```diff
@@ -167,12 +167,9 @@ impl Validatable for BitcoinConfig {
             }
         }
 
-        if self.chain_tip_polling_interval.as_secs()
-            > MAX_BITCOIN_CHAIN_TIP_POLLING_INTERVAL_SECONDS
-        {
+        if self.chain_tip_polling_interval == std::time::Duration::ZERO {
             return Err(ConfigError::Message(
-                "[bitcoin.chain_tip_polling_interval] Maximum polling interval exceeded"
-                    .to_string(),
+                "[bitcoin.chain_tip_polling_interval] Zero is not allowed".to_string(),
```

Comment:
```suggestion
                "[bitcoin.chain_tip_polling_interval] smaller then one second is not allowed".to_string(),
```

Adjusting comment according to nit above